docs/reference/buildx_rm.md

```diff
@@ -7,6 +7,13 @@ docker buildx rm [NAME]
 <!---MARKER_GEN_START-->
 Remove a builder instance
 
+### Options
+
+| Name | Description |
+| --- | --- |
+| `--builder string` | Override the configured builder instance |
+| `--keep-state` | Keep BuildKit state |
```

Comment:
so it can be reused by a new builder with the same name. Only supported by the `docker-container` driver at the moment.

This can also be in a separate block explaining the flag.